### ISSUE TITLE
Speed up the nodes startup

### ISSUE THREAD

**Issue by jrmanes** [celestiaorg/test-infra#156]
Hello team,

The following ticket is created in order to investigate the if we can use the Kubernetes' `init` containers in order to sync the data before starting the container, this will help us to speed up the process when the **celestia-app** has to sync the whole blockchain when starting.

There are some things to consider:

- [ ] We have to create a service which is running within the cluster and sync the data from the blockchain (that means, it has to be part of the blockchain, or at least being able to get the data from it) to a PVC(for instance), every X minutes to keep it updated(we should be able to adjust this parameter).
  - [ ] We could use a `Job`, `CronJob`, or develop an small service for being more flexible.
- [ ] Check if we don't damage the blockchain if we use "old" data.
- [ ] Considering the vendor lock, we have to be able to do it in the different cloud providers.
- [ ] The `init` container has to be able to get the data (wherever we keep it) and sync it to the `data` folder in the pod.
- [ ] Track the speed for having numbers 👀 

--- 

Thanks in advance,

Jose Ramon Mañes

**Comment by Bidon15** [comment]
Grooming 10/03/2023: @jrmanes can your port this to `devops` repo and add to the project board 🙏 ? And close this one

1. Save data as a snapshot
2. You can import a snapshot
3. Try existing bash scripts -> port to `Go`

**Comment by Bidon15** [comment]
And link the new issue to https://github.com/celestiaorg/devops/issues/43

**Comment by jrmanes** [comment]
moved to `devops` 👌 

https://github.com/celestiaorg/devops/issues/80

*not added to any sprint yet*


cheers!
